<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageContent</class>
 <widget class="QWidget" name="ImageContent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Conteúdo em Imagem</string>
  </property>
  <widget class="QPlainTextEdit" name="txt_duration">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>520</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancelar</string>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/i/no.png</normaloff>:/i/no.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QTimeEdit" name="time_end">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_duration">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>16</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Duração (segundos):</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>520</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/i/check.png</normaloff>:/i/check.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>25</width>
     <height>25</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_end_time">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>16</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tempo de Termino:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_begin_time">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>16</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tempo de Início:</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="time_begin">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txt_media_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>541</width>
     <height>41</height>
    </rect>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_choose_image">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Escolher Imagem</string>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/i/folder_open.png</normaloff>:/i/folder_open.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="layout_selector_widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>721</width>
     <height>411</height>
    </rect>
   </property>
  </widget>
  <zorder>layout_selector_widget</zorder>
  <zorder>txt_duration</zorder>
  <zorder>btn_cancel</zorder>
  <zorder>time_end</zorder>
  <zorder>lbl_duration</zorder>
  <zorder>btn_ok</zorder>
  <zorder>lbl_end_time</zorder>
  <zorder>lbl_begin_time</zorder>
  <zorder>time_begin</zorder>
  <zorder>txt_media_name</zorder>
  <zorder>btn_choose_image</zorder>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
